### ISSUE TITLE
Error Doing Job: Task Exception was never retrieved

### ISSUE THREAD

**Issue by mbhforum** [pyalarmdotcom/alarmdotcom#347]
This doesn't happen with 3.0.9, but does with 3.0.10.

This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/alarmdotcom/base_device.py:198
Integration: Alarm.com (documentation, issues)
First occurred: 1:32:06 PM (307 occurrences)
Last logged: 6:40:26 PM

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/pyalarmdotcomajax/devices/registry.py", line 299, in get
    return self._devices[device_id]
           ~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 37248426

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 233, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 389, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 172, in async_update_listeners
    update_callback()
  File "/config/custom_components/alarmdotcom/base_device.py", line 91, in _handle_coordinator_update
    super()._handle_coordinator_update()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 469, in _handle_coordinator_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 743, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 843, in _async_write_ha_state
    state, attr = self._async_generate_attributes()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 787, in _async_generate_attributes
    attr.update(self.extra_state_attributes or {})
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/alarmdotcom/base_device.py", line 198, in extra_state_attributes
    raw = self._controller.api.devices.get(self._adc_id).raw_attributes
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pyalarmdotcomajax/devices/registry.py", line 301, in get
    raise UnkonwnDevice(device_id) from err
pyalarmdotcomajax.exceptions.UnkonwnDevice: Unknown device ID '37248426'.

**Describe the bug**
From the GUI, whenever I try to change the state of my alarm, the above error appears in the logs when  it try to change the state of the alarm (eg: disarm to Armed Away). I am able to disarm and arm stay, but not arm away.  

**To Reproduce**
Steps to reproduce the behavior:

See above 

**Expected behavior**
A clear and concise description of what you expected to happen.

Alarm goes to right state and updates.  

**Screenshots**
N/A

**Home Assistant Version:**

2023.11.0, running inside a Docker container

**Additional context**
Add any other context about the problem here.

**Comment by Axsgranted** [comment]
I have the same issue but I'm not trying to change the state of the panel or any of the entities.  It just keeps happening at least once a minute.  My error log copied below:

> Error doing job: Task exception was never retrieved

> Traceback (most recent call last):
>   File "/usr/local/lib/python3.11/site-packages/pyalarmdotcomajax/devices/registry.py", line 299, in get
>     return self._devices[device_id]
>            ~~~~~~~~~~~~~^^^^^^^^^^^
> KeyError: 18281922
> 
> The above exception was the direct cause of the following exception:
> 
> Traceback (most recent call last):
>   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 233, in _handle_refresh_interval
>     await self._async_refresh(log_failures=True, scheduled=True)
>   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 389, in _async_refresh
>     self.async_update_listeners()
>   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 172, in async_update_listeners
>     update_callback()
>   File "/config/custom_components/alarmdotcom/base_device.py", line 91, in _handle_coordinator_update
>     super()._handle_coordinator_update()
>   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 469, in _handle_coordinator_update
>     self.async_write_ha_state()
>   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 743, in async_write_ha_state
>     self._async_write_ha_state()
>   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 843, in _async_write_ha_state
>     state, attr = self._async_generate_attributes()
>                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 787, in _async_generate_attributes
>     attr.update(self.extra_state_attributes or {})
>                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/config/custom_components/alarmdotcom/base_device.py", line 198, in extra_state_attributes
>     raw = self._controller.api.devices.get(self._adc_id).raw_attributes
>           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/usr/local/lib/python3.11/site-packages/pyalarmdotcomajax/devices/registry.py", line 301, in get
>     raise UnkonwnDevice(device_id) from err
> pyalarmdotcomajax.exceptions.UnkonwnDevice: Unknown device ID '18281922'.

**Comment by Axsgranted** [comment]
Might have figured out a connection.

The "Unknown device ID '18281922'", referenced in my logs is the same ID referenced by the scenes that got created for "home", "away", etc. statuses.

I disabled all the "scenes" the integration created and these logs appeared to stop.

**Comment by mbhforum** [comment]
That didn't get rid of the errors for me.  I disabled the scenes in the Entities section, but everytime I reload Alarm.com integration, it gets re-enabled.  How did you accomplish this?

**Comment by eliasisrael** [comment]
I am also experiencing this same issue. Anyone have any new ideas about it?

**Comment by eliasisrael** [comment]
This has appeared 977 times in my log>>


This error originated from a custom integration.

Logger: homeassistant
Source: custom_components/alarmdotcom/base_device.py:198
Integration: Alarm.com (documentation, issues)
First occurred: January 6, 2024 at 9:44:40 PM (977 occurrences)
Last logged: 7:19:34 AM

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/pyalarmdotcomajax/devices/registry.py", line 299, in get
    return self._devices[device_id]
           ~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 10169509

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 243, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 399, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 182, in async_update_listeners
    update_callback()
  File "/config/custom_components/alarmdotcom/base_device.py", line 91, in _handle_coordinator_update
    super()._handle_coordinator_update()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 479, in _handle_coordinator_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 941, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1062, in _async_write_ha_state
    state, attr, capabilities, shadowed_attr = self.__async_calculate_state()
                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1002, in __async_calculate_state
    attr.update(self.extra_state_attributes or {})
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/alarmdotcom/base_device.py", line 198, in extra_state_attributes
    raw = self._controller.api.devices.get(self._adc_id).raw_attributes
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pyalarmdotcomajax/devices/registry.py", line 301, in get
    raise UnkonwnDevice(device_id) from err
pyalarmdotcomajax.exceptions.UnkonwnDevice: Unknown device ID '10169509'.

**Comment by elahd** [comment]
This is a bug with the scenes feature. Are you all on integration version 3.0.11? Should be fixed there.

**Comment by eliasisrael** [comment]
When I try to download 3.0.11, it keeps giving me 3.0.10

**Comment by sargonas** [comment]
Same issue here... attempts to update to 3.0.11 result in having 3.0.10 after the process complets.

I also noticed that in the drop down menu for versions, 3.0.11 is the only one without a v in front of the number, and it also does not prompt me with an "update is available." I think perhaps this release was misconfigured?

**Comment by elahd** [comment]
This is fixed in v3.0.12.
